ut/m0ut.c

```diff
@@ -498,9 +499,16 @@ int main(int argc, char *argv[])
 			m0_ut_list(with_tests, yaml_output);
 		else if (list_owners)
 			m0_ut_list_owners();
-		else
+		else {
+			m0_console_printf("START Iteration: %d out of %d\n",
```

Comment:
For the first iteration this  line would display "START Iteration: 0 out of 1. Since i starts from '0'. Iteration count should start from '1' and increment. You might want to change the for loop to start with i =1 and i <= count so that the iteration number displayed makes sense.

Reply:
Yes. I considered to do so. But later, I kept it as this.
I need someone to propose "starting from 1".
:)